provider/aws.go

```diff
@@ -277,16 +281,16 @@ func newChange(action string, endpoint *endpoint.Endpoint) *route53.Change {
 	if isAWSLoadBalancer(endpoint) {
 		change.ResourceRecordSet.Type = aws.String(route53.RRTypeA)
 		change.ResourceRecordSet.AliasTarget = &route53.AliasTarget{
-			DNSName:              aws.String(endpoint.Target),
-			HostedZoneId:         aws.String(canonicalHostedZone(endpoint.Target)),
+			DNSName:              aws.String(endpoint.Targets[0]),
+			HostedZoneId:         aws.String(canonicalHostedZone(endpoint.Targets[0])),
 			EvaluateTargetHealth: aws.Bool(evaluateTargetHealth),
 		}
 	} else {
 		change.ResourceRecordSet.Type = aws.String(suitableType(endpoint))
 		change.ResourceRecordSet.TTL = aws.Int64(recordTTL)
 		change.ResourceRecordSet.ResourceRecords = []*route53.ResourceRecord{
 			{
-				Value: aws.String(endpoint.Target),
+				Value: aws.String(endpoint.Targets[0]),
```

Comment:
I think this needs to iterate over the Targets available and add them each as a separate ResourceRecord.